### PR TITLE
Optimize parameter acquisition method in python script

### PR DESCRIPTION
**Pull request by ShenYuhan**
Optimize parameter acquisition method in python script

### CODE REVIEW

File: visualdl/io/bfile.py [hunk 1]
```diff
@@ -31,6 +31,11 @@ def register_filesystem(self, path, filesystem):
         self._register_factories.update({path: filesystem})
 
     def get_filesystem(self, path):
+        if 'hdfs://' in path and "hdfs" not in self._register_factories.keys():
```
Comment (PeterPanZH) [inline visualdl/io/bfile.py]: `path.startswith('hdfs://')` makes much sense?

Comment (ShenYuhan) [reply visualdl/io/bfile.py]: Professional advice, i will modify it.

File: visualdl/io/bfile.py [hunk 2]
```diff
@@ -31,6 +31,11 @@ def register_filesystem(self, path, filesystem):
...
+            try:
+                default_file_factory.register_filesystem("hdfs", HDFileSystem())
+            except hdfs.util.HdfsError:
+                raise ValueError("Please initialize `~/.hdfscli.cfg` for HDFS.")
```
Comment (nepeplwu) [inline visualdl/io/bfile.py]: 这种情况下抛RuntimeError

Comment (ShenYuhan) [reply visualdl/io/bfile.py]: 好

**Pull request closed without merging**